src/components/main.tsx

```diff
@@ -39,6 +40,7 @@ export const Main = () => {
     <Box w="100%" h="100svh" position="relative" overflow="none">
       <Header option={option} setOption={setOption} />
       <Panel choiceMoonquake={choiceMoonquake} />
+      {isMap || <YearSlider option={option} setOption={setOption} />}
```

Comment:
なぜor？
やるなら↓の方が良さそうかも？(意味的には同じだけどreactでこの書き方でorを使う文化があまりなさそうなので)

```suggestion
      {!isMap && <YearSlider option={option} setOption={setOption} />}
```